src/DataProtection/DataProtection/src/KeyManagement/KeyManagementOptions.cs

```diff
@@ -111,7 +111,7 @@ internal static TimeSpan MaxServerClockSkew
     /// <remarks>
     /// Settable for testing.
     /// </remarks>
-    internal TimeSpan DefaultKeyResolverRetryDelay { get; set; } = TimeSpan.FromMilliseconds(200);
+    internal TimeSpan DefaultKeyResolverRetryDelay { get; set; } = TimeSpan.FromMilliseconds(200.0);
```

Comment:
nit: an odd thing to include in a PR; was this to make a warning happy?

Reply:
yep, rollbacked it